charts/alfresco-ai-transformer/templates/config-ai-transform-pipelines.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  {{- $cmCtx := dict "Values" (dict "nameOverride" "ai-pipelines") "Chart" .Chart "Release" .Release }}
+  {{- $cmCtx := dict "Values" (dict "fullnameOverride" "ai-transform-pipelines") "Chart" .Chart "Release" .Release }}
```

Comment:
it still make sense to use the named template with fullnameOverride? just hardcoded it then (or define a new value)

Reply:
not much indeed